database/src/models/account.rs

```diff
@@ -221,4 +223,33 @@ impl Account {
             .execute(&**db_conn)
             .and(Ok(()))
     }
+
+    pub fn display_name_or_username(&self) -> &str {
+        self.display_name.as_ref().unwrap_or(&self.username)
+    }
+}
+
+impl AsRef<Account> for Account {
```

Comment:
hmmmm i'm confused as to why you're impling this; isn't `some_account.as_ref()` going to be just the same as `&some_account` then? unless this is to work around some idiosyncrasy of askama.

Reply:
That’s pretty much exactly it - ‘&’ in a Askama template causes a parse failure, so ‘as_ref’ required